src/protocols/secure_channel/MessageCounterManager.cpp

```diff
@@ -433,13 +437,11 @@ void MessageCounterSyncMgr::HandleMsgCounterSyncResp(Messaging::ExchangeContext
 
     if (exchangeContext != nullptr)
```

Comment:
How can `exchangeContext` be nullptr here?  We can't reach this line without having dereferenced it.